src/main/java/com/stripe/net/LiveStripeResponseGetter.java

```diff
@@ -523,8 +534,12 @@ private static StripeResponse rawRequest(
       ApiResource.RequestMethod method, String url, Map<String, Object> params,
       Class<T> clazz, ApiResource.RequestType type, RequestOptions options)
       throws StripeException {
+    long requestStart = System.currentTimeMillis();
```

Comment:
Some of the spacing in this file doens't really line up, can you make sure the spacing is correct? (if you use intellij the standard formatter will probably work just fine)

Reply:
I agree that this looks a little off, but I followed the spacing that was already being used. Reformatting with IntelliJ's formatter would end up changing the whole file.

Reply:
Can we call this one `requestStartMS` (i.e. add the `MS` prefix here as well)? It's not a time object like other languages.